GDJS/package-lock.json

```diff
@@ -1,3806 +1,8 @@
 {
   "name": "gdevelop-javascript-platform",
   "version": "0.0.0",
-  "lockfileVersion": 2,
```

Comment:
I don't think this downgrade of the lockfile, please remove it from the PR (`git reset master ./GDJS/package-lock.json && git restore ./GDJS/package-lock.json`)

Reply:
(Ditto for newIDE/app)